tests/parallel/ExportTimeseries.cpp

```diff
@@ -26,7 +26,8 @@ BOOST_AUTO_TEST_CASE(ExportTimeseries)
   }
 
   double time = 0.0;
-  double dt   = interface.initialize();
+  interface.initialize();
+  double dt = interface.getMaxTimeStepSize();
```

Comment:
I we want to shift this down (again "right before the `min`"), I would rather wait for #1603, because I will have to touch all test in either of the two PRs.